modules/rostests/apitests/crt/crtdll_crt_apitest.cmake

```diff
@@ -260,7 +260,6 @@ list(APPEND SOURCE_CRTDLL
 #    _scalb.c
 #    _searchenv.c
 #    _seterrormode.c
-#    _setjmp.c
```

Comment:
Why do you make it `ARCH` specific/duplicate? (Everywhere)

Reply:
Because ARM64 is not supported yet.

Reply:
What actual failure would this cause on ARM64 (compilation)?
Otherwise, I would say "keep it as it is meant to be, for the time being".

Reply:
AMD64 is not ARM64.